.travis/build-ubuntu_14_04.sh

```diff
@@ -117,6 +117,8 @@ cd ..
 
 $CC --version
 $CXX --version
+
+export PKG_CONFIG_PATH="$PWD/libs/lib/pkgconfig"
```

Comment:
This should be the script dir shouldn't it? You can get that reliable with: `SCRIPT_DIR=$(dirname `readlink -f -- $0`)`